broadcasting.md

```diff
@@ -247,6 +255,47 @@ npm run dev
 > **Note**  
 > To learn more about compiling your application's JavaScript assets, please consult the documentation on [Vite](/docs/{{version}}/vite).
 
+Additional supported features :
+
+*1. Modify private/presence channel capability. Default: Full capability*
+- Channel access control rights are granted for each individual user separately using `ably-capability`.
+- It defines list of access claims as per [Channel Capabilities](https://ably.com/docs/core-features/authentication#capability-operations).
+
+```php
+  // file - routes/channels.php
+  // User authentication is allowed for private/presence channel returning truthy values and denied for falsy values.
+  
+  // for private channel
```

Comment:
```suggestion
  // for private channels
```

Reply:
Actually, we are defining it for a single private channel named `channel`. so I kept it singular.

Reply:
`Broadcast::channel('channel1', function ($user) {`

Reply:
Yeah that makes sense but it's still bad grammar. I think if you put "for private channels" it reads as if you're saying "this is how to configure capabilities for private channels" so it makes sense even if the example just shows one channel. If you insist on keeping it singular, the grammatically correct way of writing it would be "for a presence channel"

Reply:
done